File: src/main/java/io/vproxy/vfx/ui/scene/VSceneGroupInitParams.java
package io.vproxy.vfx.ui.scene;

public class VSceneGroupInitParams {
    public boolean gradientCover = false;
    public boolean useClip = true;

    public VSceneGroupInitParams setGradientCover(boolean gradientCover) {
        this.gradientCover = gradientCover;
        return this;
    }

    public VSceneGroupInitParams setUseClip(boolean useClip) {
        this.useClip = useClip;
        return this;
    }
}

File: src-ts/src/ui/scene/VSceneGroupInitParams.ts
export class VSceneGroupInitParams {
  gradientCover = false;
  useClip = true;

  setGradientCover(v: boolean): this {
    this.gradientCover = v;
    return this;
  }
  setUseClip(v: boolean): this {
    this.useClip = v;
    return this;
  }
}
